cluster/kube-up.sh

```diff
@@ -54,4 +54,13 @@ if [[ "${ENABLE_CLUSTER_DNS}" == "true" ]]; then
       | "${KUBE_ROOT}/cluster/kubectl.sh" create -f -
 fi
 
+# If logging with Fluentd to Elasticsearch is enabled then create pods
+# and services for Elasticsearch (for ingesting logs) and Kibana (for
+# viewing logs).
+if [[ "${ENABLE_NODE_LOGGING-}" == "true" ]]; then
```

Comment:
I think it's worthwhile to split ENABLE and setup for on-node logging infra from in-cluster logging infra.